README.md

```diff
@@ -77,48 +77,37 @@ In the case that you'd like to serve and test a static build of the documentatio
 yarn docs:build
 ```
 
-# Updating Spectrum CSS
```

Comment:
Removed this section and bumped the following heading levels (should only have 1 h1 on a page)